### ISSUE TITLE
Button: with `mdc-button--unelevated` class does not get "our own" transitions styles.

### ISSUE THREAD

**Issue by Kiarokh** [Lundalogik/lime-elements#2003]
https://user-images.githubusercontent.com/35954987/210080625-5f169aaf-c69f-4bcc-9996-5ccdb30daf29.mov

**Comment by Kiarokh** [comment]
All of our buttons are elevated. We don't have unelevated buttons (except in icon-button, but that's another story).
So the question is whether we should have this class at all, or not?
![image](https://user-images.githubusercontent.com/35954987/210081028-677a7379-249e-4115-b621-2b3d547a590c.png)

**Comment by adrianschmidt** [comment]
> So the question is whether we should have this class at all, or not?

No, that seems like there's been a mistake made somewhere… I think the buttons were once not elevated, and when changing them to be elevated, instead of changing which class was used, the styles for the `mdc-button--unelevated` class was changed? 🤷‍♂️

I've traced the addition of `mdc-button--unelevated` only for primary buttons back to commit 95a8e58c09e4164360e03e941fed1e2cf804f7e0, where I seem to have misinterpreted the meaning of `.classList.add('mdc-button--unelevated', 'primary');`… unless I changed that behaviour intentionally, but I don't really see why I would have.

Either way, feel free to remove that class! 🙂
